vroom/main.vroom

```diff
@@ -185,8 +220,8 @@ them are available, the message will show a line for each.
   :let fake_format2 = copy(fake_format)
   :let fake_format2.name = 'fake-format2'
   :let fake_format2.setup_instructions = 'LMGTFY'
-  :call codefmtlib#AddFormatter(fake_format2)
+  :call maktaba#extension#GetRegistry('codefmt').AddExtension(fake_format2)
 
   :FormatCode
-  ~ Formatter "fake-format" is not available. Setup instructions: RTFM
   ~ Formatter "fake-format2" is not available. Setup instructions: LMGTFY
+  ~ Formatter "fake-format" is not available. Setup instructions: RTFM
```

Comment:
codefmtlib did in-order registration (with two priorities), and then codefmt did first-wins over [external + internal].

First-wins is much more convenient for callers, so we retained that in `maktaba#extensions`, but in-order registration doesn't let you override an existing registration, so `maktaba#extensions` inserts at the _start_ of the respective lists.

That means that if you register formatter `foo` and later register `bar`, we now check `bar` first to give it a chance of overriding `foo`.  That normally doesn't matter, but in this case (and clang-format/jsbeautify above), it does, as both formatters handle the same filetypes.